docs/reference/commandline/volume_create.md

```diff
@@ -0,0 +1,42 @@
+<!--[metadata]>
++++
+title = "volume create"
+description = "The volume create command description and usage"
+keywords = ["volume, create"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# volume create
+
+    Usage: docker volume create [OPTIONS]
+
+    Create a volume
+
+    -d, --driver=local    Specify volume driver name
+    --help=false          Print usage
+    --name=               Specify volume name
+    -o, --opt=map[]       Set driver specific options
+
+Creates a new volume that containers can can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
+
+  $ docker volume create --name hello
+  hello
+  $ docker run -d -v hello:/world busybox ls /world
+
+The mount is created inside the container's `/src` directory. Docker does not support relative paths for mount points inside the container. 
```

Comment:
I do not understand this. Executing the example, I see that the volume is mounted as `/world`. There is no  `/src` in the container.